Classes/Service/AuthenticationService.php

```diff
@@ -628,7 +627,13 @@ protected function applyMapping(string $table, array $oidc, array $typo3, array
             $feSim->cleanupTSFE();
         }
 
-        return $out;
+        $event = new AuthenticationProcessMappingEvent($typo3User, $out);
```

Comment:
It might be nice to pass a readonly `$oidc` here as well. Maybe the mapping depends on the kind of user. Maybe even `$table` and `$request`.

Reply:
You were faster than me. I was just thinking about more context here